api/v1alpha1/condition_consts.go

```diff
@@ -49,3 +49,27 @@ const (
 	// DeletedReason (Severity=Info) documents a condition not in Status=True because the underlying object was deleted.
 	DeletedReason = "Deleted"
 )
+
+// Conditions related to the VirtualMachineImages
+const(
+	// VirtualMachineImageOSTypeSupportedCondition denotes that the OS type in the VirtualMachineImage object is
+	// supported by VMService. A VirtualMachineImageOsTypeSupportedCondition is marked true:
+	// - If OS Type is of Linux Family
+	// - If OS Type is supported by hosts in the cluster
+	VirtualMachineImageOSTypeSupportedCondition ConditionType = "VirtualMachineImageOSTypeSupported"
+
+	// VirtualMachineImageV1Alpha1CompatibleCondition denotes image compatibility with VMService. VMService expects
+	// VirtualMachineImage to be prepared by VMware specifically for VM Service v1alpha1.
```

Comment:
"VM Service" -> "VMService" for consistency?

Reply:
I had changed all of these to VMService, rebase probably messed this up. Thanks for pointing it out. Will fix it